conda/base/context.py

```diff
@@ -891,13 +891,7 @@ def channels(self):
             if not self.override_channels_enabled:
                 from ..exceptions import OperationNotAllowed
 
-                raise OperationNotAllowed(
-                    dals(
-                        """
-                Overriding channels has been disabled.
-                """
-                    )
-                )
+                raise OperationNotAllowed("Overriding channels has been disabled.")
```

Comment:
Changes like these are so weird, will this change the CLI rendering significantly?

Reply:
For this one no I don't think so, this was just poorly formatted to start